news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: We are working on a new front office theme
+subtitle: Here are some more details about it
+date: 2022-01-13
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.0]
```

Comment:
Make sure tags exist
I'm not sure

Ref https://github.com/PrestaShop/prestashop.github.io/issues/813
https://github.com/PrestaShop/prestashop.github.io/blob/master/_data/tags.yml

Reply:
Should we use the slug or the name?

Reply:
Slug so it has to be either 8.0.0 or 8.x